src/MagnumPlugins/DrWavAudioImporter/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+#
+#   This file is part of Magnum.
+#
+#   Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016
+#             Vladimír Vondruš <mosra@centrum.cz>
+#   Copyright © 2016 Alice Margatroid <loveoverwhelming@gmail.com>
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a
+#   copy of this software and associated documentation files (the "Software"),
+#   to deal in the Software without restriction, including without limitation
+#   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell copies of the Software, and to permit persons to whom the
+#   Software is furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included
+#   in all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#   DEALINGS IN THE SOFTWARE.
+#
+
+find_package(Magnum REQUIRED Audio)
+
+if(BUILD_STATIC)
+    set(MAGNUM_DRWAVAUDIOIMPORTER_BUILD_STATIC 1)
+endif()
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
+               ${CMAKE_CURRENT_BINARY_DIR}/configure.h)
+
+set(DrWavAudioImporter_SRCS
+    DrWavImporter.cpp)
+
+set(DrWavAudioImporter_HEADERS
+    DrWavImporter.h)
+
+# Objects shared between plugin and test library
+add_library(DrWavAudioImporterObjects OBJECT
+    ${DrWavAudioImporter_SRCS}
+    ${DrWavAudioImporter_HEADERS})
+target_include_directories(DrWavAudioImporterObjects PUBLIC
+    $<TARGET_PROPERTY:Magnum::Audio,INTERFACE_INCLUDE_DIRECTORIES>
+    ${PROJECT_SOURCE_DIR}/src
+    ${PROJECT_BINARY_DIR}/src)
+target_compile_definitions(DrWavAudioImporterObjects PRIVATE "DrWavAudioImporterObjects_EXPORTS")
+# Include the stb_*.h files as a system directory to supress warnings
```

Comment:
Very minor thing: this still references the `stb_*` files, should be `dr_*` I think ;)